### ISSUE TITLE
Dropdown buttons when browsing exhibits don't have enough color contrast

### ISSUE THREAD

**Issue by zaymasood** [pulibrary/dpul#1228]
### Errors in spec/features/browse_spec.rb
     Browsing exhibits when logged in as a site admin. when browsing all exhibit items complies with WCAG
          color-contrast: Elements must have sufficient color contrast (serious)
               https://dequeuniversity.com/rules/axe/4.3/color-contrast?application=axeAPI
               
                    Selector: #sort-dropdown > .dropdown-toggle[name="button"][data-toggle="dropdown"]
                    HTML: <button name="button" type="submit" class="btn btn-outline-secondary dropdown-toggle" aria-expanded="false" data-toggle="dropdown" data-bs-toggle="dropdown">Sort<span class="d-none d-sm-inline"> by Relevance</span><span class="caret"></span></button>
                    Fix any of the following:
                    - Element has insufficient color contrast of 4.26 (foreground color: #6c757d, background color: #f5f4f1, font size: 12.0pt (16px), font weight: bold). Expected contrast ratio of 4.5:1
               
                    Selector: #sort-dropdown > .dropdown-toggle[name="button"][data-toggle="dropdown"] > .d-none.d-sm-inline
                    HTML: <span class="d-none d-sm-inline"> by Relevance</span>
                    Fix any of the following:
                    - Element has insufficient color contrast of 4.26 (foreground color: #6c757d, background color: #f5f4f1, font size: 12.0pt (16px), font weight: bold). Expected contrast ratio of 4.5:1
               
                    Selector: #per_page-dropdown > .dropdown-toggle[name="button"][data-toggle="dropdown"]
                    HTML: <button name="button" type="submit" class="btn btn-outline-secondary dropdown-toggle" aria-expanded="false" data-toggle="dropdown" data-bs-toggle="dropdown">10<span class="sr-only visually-hidden"> per page</span><span class="d-none d-sm-inline"> per page</span><span class="caret"></span></button>
                    Fix any of the following:
                    - Element has insufficient color contrast of 4.26 (foreground color: #6c757d, background color: #f5f4f1, font size: 12.0pt (16px), font weight: bold). Expected contrast ratio of 4.5:1
               
                    Selector: #per_page-dropdown > .dropdown-toggle[name="button"][data-toggle="dropdown"] > .d-none.d-sm-inline
                    HTML: <span class="d-none d-sm-inline"> per page</span>
                    Fix any of the following:
                    - Element has insufficient color contrast of 4.26 (foreground color: #6c757d, background color: #f5f4f1, font size: 12.0pt (16px), font weight: bold). Expected contrast ratio of 4.5:1
###
     Browsing exhibits when logged in as a site admin. when browsing all exhibit items in a saved search complies with WCAG
          color-contrast: Elements must have sufficient color contrast (serious)
               https://dequeuniversity.com/rules/axe/4.3/color-contrast?application=axeAPI
               
                    Selector: #sort-dropdown > .dropdown-toggle[name="button"][data-toggle="dropdown"]
                    HTML: <button name="button" type="submit" class="btn btn-outline-secondary dropdown-toggle" aria-expanded="false" data-toggle="dropdown" data-bs-toggle="dropdown">Sort<span class="d-none d-sm-inline"> by Relevance</span><span class="caret"></span></button>
                    Fix any of the following:
                    - Element has insufficient color contrast of 4.26 (foreground color: #6c757d, background color: #f5f4f1, font size: 12.0pt (16px), font weight: bold). Expected contrast ratio of 4.5:1
               
                    Selector: #sort-dropdown > .dropdown-toggle[name="button"][data-toggle="dropdown"] > .d-none.d-sm-inline
                    HTML: <span class="d-none d-sm-inline"> by Relevance</span>
                    Fix any of the following:
                    - Element has insufficient color contrast of 4.26 (foreground color: #6c757d, background color: #f5f4f1, font size: 12.0pt (16px), font weight: bold). Expected contrast ratio of 4.5:1
               
                    Selector: #per_page-dropdown > .dropdown-toggle[name="button"][data-toggle="dropdown"]
                    HTML: <button name="button" type="submit" class="btn btn-outline-secondary dropdown-toggle" aria-expanded="false" data-toggle="dropdown" data-bs-toggle="dropdown">10<span class="sr-only visually-hidden"> per page</span><span class="d-none d-sm-inline"> per page</span><span class="caret"></span></button>
                    Fix any of the following:
                    - Element has insufficient color contrast of 4.26 (foreground color: #6c757d, background color: #f5f4f1, font size: 12.0pt (16px), font weight: bold). Expected contrast ratio of 4.5:1
               
                    Selector: #per_page-dropdown > .dropdown-toggle[name="button"][data-toggle="dropdown"] > .d-none.d-sm-inline
                    HTML: <span class="d-none d-sm-inline"> per page</span>
                    Fix any of the following:
                    - Element has insufficient color contrast of 4.26 (foreground color: #6c757d, background color: #f5f4f1, font size: 12.0pt (16px), font weight: bold). Expected contrast ratio of 4.5:1

**Comment by thanyabegum** [comment]
Duplicate of #1218
